source/documentation/contribute-to-rdoproject.html.md

```diff
@@ -0,0 +1,92 @@
+---
+author: chandankumar
+title: Let's Contribute to RDO Project
+---
+
+# Let's Contribute to RDO Project
+
+So contributing to RDO Project is hard, I think no! we need to follow some steps
+and we are done towards making first contribution to RDO Project.
+
+## Let's get prerequities ready.
+
+### Join the RDO Mailing List
+* Go to [RDO mailing list](https://www.redhat.com/mailman/listinfo/rdo-list) and enter your email
```

Comment:
I think to avoid duplicate content it's better to reference https://www.rdoproject.org/community/mailing-lists/ so that we need to update only one place when anything is changed.